README.md

```diff
@@ -1,32 +1,127 @@
+* Excellent script editor
+* 2D and 3D mapper with autowalk
+* Powerful and simple to use GMCP
+* Discord integration
+* Completely themable
+* Open-source and Free
+* Cross platform: Windows, macOS and Linux
+    * full compatibility: run same scripts on any of the above
 
-## Easy to use client
+<details>
+  <summary>Explain?</summary>
+  
+### Easy to use client
 
-We’re big on usability, and as such, creating an easy to use client and interface is one of the defining goals of the project. This applies to both the power users and plain gamers – everyone will feel at home with Mudlet, without having to waste too much time figuring out how to do something.
+We’re big on usability, and as such, creating an easy to use client and interface is one of the defining goals of the project. This applies to both the power users and usual gamers – everyone will feel at home with Mudlet, without having to waste too much time figuring out how to do something.
 
-## Designed for speed
+### Designed for speed
 
 Mudlet is designed and built to be very fast and efficient right from the start. Its scripting engine is designed to scale to large systems without bogging down – and the text display is designed to handle thousands of lines in under a second. All in all, we are very serious about Mudlet being quick – and take all measures to make it so.
 
-## Powerful Scripting
+### Powerful Scripting
 
 Mudlet features a scripting framework using [Lua](https://www.lua.org/) – a small, fast and efficient scripting language. This allows Mudlet to leverage the existing community and large ecosystem of existing packages for Lua without the many drawbacks of creating a Mudlet specific scripting language. Best of all, the Lua API is seamlessly integrated in Mudlet and shared by all aliases, triggers, scripts, keybindings, buttons and other Mudlet components.
 
-## Cross-Platform love
+### Cross-Platform love
+
+We believe in making Mudlet available to people on all major platforms, and we work on keeping cross-platform compatibility right from start. Mudlet is available on Linux (both 32bit and 64bit), Windows, and macOS; you may be able to run it on additional platforms as well.
+</details>
+
+## Download
+
+Download for free from [mudlet.org](https://www.mudlet.org/download/)
+
+## How To Use
+
+1. Download and double-click on Mudlet to run
+1. Pick an existing game and hit `Connect`
+1. ... or play one of your choosing:
+    1. Click `New`
+    1. Enter `Server address` and `Port` of your game (found on the game's website, e.g. `mygame.com` and `2003`)
+    1. Pick a `Profile name`
+    1. Hit `Connect`
+
+## Roadmap
+
+See the [project's roadmap](https://www.mudlet.org/2018/02/mudlet-roadmap-2018) to get an idea of where it's headed, as well as contribute!
+
+## Credits
+
+Author: [Mudlet Makers](https://github.com/Mudlet/Mudlet/graphs/contributors). Join in, contributions are welcome!
+
+This software wouldnt've been possible without these open source packages:
```

Comment:
"wouldnt've" -> "wouldn't have" or even better "would not have"

Reply:
This word is fine...

Reply:
Never seen it written before. In speech, ok, maybe.
Looking at the wikitionary now, it is indeed listed, but only as colloquial:
https://en.wiktionary.org/wiki/wouldn%27t%27ve

Reply:
Well we're not writing an ISO specification here so it's OK :P